service/src/main/kotlin/fi/espoo/evaka/shared/FeatureConfig.kt

```diff
@@ -86,4 +86,7 @@ data class FeatureConfig(
 
     /** Controls whether permission to share is required for curriculum documents */
     val curriculumDocumentPermissionToShareRequired: Boolean,
+
+    /** Optionally change the number of days citizens can move daycare start. Default is 14 days or 2 weeks. */
+    val requestedStartUpperLimit: Int = 14
```

Comment:
I think having a default is a bad idea. Many of these config variables could have a default, but none have.

Reply:
I meant that the default should be removed 🙈

Reply:
I know, I thought I did 🙈 😂